### PR TITLE
[SIG 3783] area map selected incident

### PR DESCRIPTION
**Pull request by samuelleeuwenburg**
Adds selected incident for the area map

### CODE REVIEW

File: src/signals/incident-management/containers/AreaContainer/AreaContainer.tsx [hunk 1]
```diff
@@ -32,13 +33,21 @@ export const AreaContainer: FunctionComponent = () => {
...
     getArea(
       `${configuration.INCIDENT_PRIVATE_ENDPOINT}${id}/context/near/geography`
     )
-  }, [getArea, getIncident, id])
+
+    if (selection) {
+      getSelectedIncident(
+        `${configuration.INCIDENT_PRIVATE_ENDPOINT}${selection.properties.id}`
+      )
+    }
```
Comment (MauritsR) [inline src/signals/incident-management/containers/AreaContainer/AreaContainer.tsx:49]: Better to move this into a separate `useEffect`. Now, every time the selection changes, the two calls above will also be triggered.

File: src/signals/incident-management/containers/AreaContainer/AreaContainer.tsx [hunk 2]
```diff
@@ -49,9 +58,19 @@ export const AreaContainer: FunctionComponent = () => {
 
   if (!area?.features || !incident) return null
 
+  const sidebar =
+    selection && selectedIncident ? (
+      <IncidentDetail
```
Comment (MauritsR) [inline src/signals/incident-management/containers/AreaContainer/AreaContainer.tsx:63]: There's no feedback to the user that data is being fetched. Probably better to show a spinner in this component (check with UX).

File: src/signals/incident-management/containers/AreaContainer/components/IncidentDetail/IncidentDetail.tsx [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  margin-top: ${themeSpacing(4)};
+`
+
+const SectionTitle = styled.div`
+  color: ${themeColor('tint', 'level5')};
+`
+
+const Status = styled.div<{ isEnded: boolean }>`
+  font-family: Avenir Next LT W01 Demi, arial, sans-serif;
+  color: ${(p) => themeColor('support', p.isEnded ? 'valid' : 'invalid')};
```
Comment (MauritsR) [inline src/signals/incident-management/containers/AreaContainer/components/IncidentDetail/IncidentDetail.tsx:46]:
```suggestion
  color: ${({ isEnded }) => themeColor('support', isEnded ? 'valid' : 'invalid')};
```

File: src/signals/incident-management/containers/AreaContainer/components/IncidentDetail/IncidentDetail.tsx [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+  onBack,
+  incident,
+}) => {
+  const isParent = Boolean(incident._links['sia:children'])
+
+  return (
+    <Wrapper>
+      <Section>
+        <BackLink to="#" onClick={() => onBack()}>
+          {' '}
```
Comment (MauritsR) [inline src/signals/incident-management/containers/AreaContainer/components/IncidentDetail/IncidentDetail.tsx:59]: Not sure if this is intentional

Comment (samuelleeuwenburg) [reply src/signals/incident-management/containers/AreaContainer/components/IncidentDetail/IncidentDetail.tsx]: nope it wasn't, my editor is very space aware

**Review by MauritsR — APPROVED** [review]

**Pull request merged**